modules/aruco/test/test_arucodetection.cpp

```diff
@@ -36,8 +36,9 @@ or tort (including negligence or otherwise) arising in any way out of
 the use of this software, even if advised of the possibility of such damage.
 */
 
-
+#include <opencv2/core/utils/logger.defines.hpp>
 #include "test_precomp.hpp"
+#include "../samples/samples_utility.hpp"
```

Comment:
Test should not rely on samples code.
Samples code should not be over-complicated by tests specific.

Reply:
I need function from `samples_utility.hpp` to advoid code duplication in tests.

Reply:
Also some ArUco samples just don't work (in 3.4/4.x).Maybe using functions in a test will allow CI to check workability of samples.

I understand, that "test should not rely on samples code". But I don't know how check workability of samples in this case.

Reply:
It makes sense (using `cv::FileNode` instead of `std::string`).

Reply:
> `FileNode node`

const reference.

---

> `Ptr<FileStorage>&`

FileNode instead of FileStorage as mentioned before.

Reply:
fixed